src/config.rs

```diff
@@ -575,6 +576,15 @@ fn load_global_config_file() -> Result<(PathBuf, Ini)> {
     }
 }
 
+fn get_failed_cli_config_load_msg(file_desc: &str) -> String {
```

Comment:
```suggestion
fn failed_config_load_message(file_desc: &str) -> String {
```

Small nitpick; I would remove the words `get` and `cli` to make the name more concise, but expand `msg` to `message`, so we use full words ideally rather than arbitrary shortenings.

Perhaps, you could also consider adding a documentation comment to state that this is the error message we print when we fail to load the CLI configuration.